LocalGoodwillOnline/src/java/com/mycompany/controllers/PhotoCameraController.java

```diff
@@ -348,6 +348,6 @@ public void deletePhoto() {
      */
     public String redo() {
         filename = "";
-        return "ChangePhoto?faces-redirect=true";
+        return "/ChangePhoto?faces-redirect=true";
```

Comment:
was this a problem?

Reply:
i don't think it was, but it doesn't hurt to have anyways. to create reserved i changed CRUD operations to be in folders, so alot of links broke because they assumed all xhtml files were in teh same directory.